src/journal-remote/journal-gatewayd.c

```diff
@@ -641,6 +641,66 @@ static int request_handler_fields(
         return MHD_queue_response(connection, MHD_HTTP_OK, response);
 }
 
+static int output_field_name(FILE *f, OutputMode m, const char *d) {
```

Comment:
our coding style suggests returning negative error numbers of failure, i.e. `return -errno` if you so want. hence it's not OK to just do `return fprintf()`